packages/cli/src/commands/deploy.ts

```diff
@@ -5,6 +5,30 @@ import { BoosterConfig, Logger } from '@boostercloud/framework-types'
 import { Script } from '../common/script'
 import Brand from '../common/brand'
 import { logger } from '../services/logger'
+import { exec } from 'child-process-promise'
+import { wrapExecError } from '../common/errors'
+import * as path from 'path'
+
+
+let skipRestoreDependencies = false;
```

Comment:
I believe you can delete this global variable and pass it as a parameter (keep reading)